Source/Krypton Components/Krypton.Toolkit/Controls Visuals/VisualMessageBoxForm.cs

```diff
@@ -667,6 +668,31 @@ private void AnyKeyDown(object sender, KeyEventArgs e)
 
         private void ShowCloseButton(bool? showCloseButton) => CloseBox = showCloseButton ?? true;
 
+        private void SetDefaultButton(KryptonMessageBoxDefaultButton defaultButton)
+        {
+            switch (defaultButton)
+            {
+                case KryptonMessageBoxDefaultButton.Button1:
+                    AcceptButton = _button1;
+                    _button1.Focus();
```

Comment:
@PWagner1 

You could also adjust only these here since the focus() method doesn't work and use Select() instead?

That should make the other adjustment I suggested redundant...

Reply:
Method is unused, so removing it...